lib/topology_lib_iperf/library.py

```diff
@@ -102,11 +120,18 @@ def client_start(
     action is taken for that parameter (left "as-is").
     :param enode: Engine node to communicate with.
     :type enode: topology.platforms.base.BaseNode
-    :param server_ip: Server's IP address in the form ``'192.168.1.10'``
-    :param int port: iperf port to be open
-    :param int interval: interval for iperf server to check
-    :param int time: the time iperf client will be running
+    :param server_ip: Server's IP address in the form ``'192.168.1.10'``.
+    :param int port: iperf port to be open.
+    :param int interval: interval for iperf server to check.
+    :param int time: the time iperf client will be running.
     :param bool udp: If it is UDP or TCP. Default is False for TCP.
```

Comment:
Will change to udp=False to match the other functions. The default comment I think still adds value here because the oppositve of UDP (TCP) may not be so obvious in meaning.